prow/scripts/cluster-integration/kyma-upgrade-gardener-azure.sh

```diff
@@ -181,15 +182,16 @@ function installKyma() {
     date
     # shellcheck disable=SC1090
     "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/create-azure-event-hubs-secret.sh
-    cat "${EVENTHUB_SECRET_OVERRIDE_FILE}" >> installer-config-azure-eventhubs.yaml.tpl
```

Comment:
could we add some comment here to tell people where this file is generated (create-azure-event-hubs-secret.sh)

Reply:
Added a note